src/pages/cart-dialog/cart-dialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Dialog, IconButton } from "@mui/material";
+import { FC } from "react";
+import { initState } from "../../reducer/navbar-reducer";
+import { backgroundColor } from "../../state/state";
+import DrawerCartBox from "../cart-drawer/drawer-cart-box/drawer-cart-box";
+import CloseOutlinedIcon from "@mui/icons-material/CloseOutlined";
+
+interface CardDialogInterface {
```

Comment:
Sometimes you use `Model` here I see `Interface`. Be consist and use only one form. The best way would be to avoid that